controllers/controllers.js

```diff
@@ -55,10 +58,17 @@ exports.getArticleComments = (req, res, next) => {
     .catch(next);
 };
 
+// PATCHs...
 exports.patchArticleById = (req, res, next) => {
   updateArticleById(req.params.article_id, req.body)
-    .then((article) => {
-      res.status(200).send({ article });
-    })
+    .then((article) => res.status(200).send({ article }))
+    .catch(next);
+};
+
+// DELETEs...
+exports.deleteCommentById = (req, res, next) => {
+  return checkExists('comments', 'comment_id', req.params.comment_id)
```

Comment:
No explicit need to check for the existence here - the rows property of the result produced by the DELETE query will be empty if the comment didn't exist